core-plugins/widgets/AzureBlobStore-batchsource.json

```diff
@@ -8,6 +8,11 @@
       "properties": [
         {
           "widget-type": "textbox",
+          "label": "Name for Dataset",
```

Comment:
What is label? It should not be "Name for Dataset", right? Should it be just "Reference Name"?